bindings/nostr-js/src/nips/nip53.rs

```diff
@@ -0,0 +1,259 @@
+// Copyright (c) 2023-2024 Rust Nostr Developers
+// Distributed under the MIT software license
+
+use std::ops::Deref;
+
+use nostr::nips::nip53::{LiveEvent, LiveEventHost, LiveEventStatus};
+use nostr::{ImageDimensions, PublicKey, UncheckedUrl};
+use wasm_bindgen::prelude::*;
+
+use crate::event::tag::JsImageDimensions;
+use crate::key::JsPublicKey;
+
+#[wasm_bindgen(js_name = Image)]
+pub struct Image {
+    url: String,
+    dimensions: Option<JsImageDimensions>,
```

Comment:
As written above, this should be public and should implement `Image::new` to make easier `Image` construction (the same in other part of codes)